main.py

```diff
@@ -211,9 +217,14 @@ def run(self):
                 self.sprites.draw(self.screen)
 
             pygame.display.flip()
-            self.clock.tick(30)
+            self.clock.tick(FPS)
 
 
 if __name__ == "__main__":
     g = Flappy()
+    # GAME_SIZE = (1920, 900)
+    # GAME_SIZE = (1024, 768)
+    # GAME_SIZE = (684, 600)
```

Comment:
You can delete these.

Reply:
I left them in for more convient testing outside of sugar env, is it not good practice?

Reply:
As they're for testing, they shouldn't be included in your commit as you're done with testing and have settled on what to use.